tests/functional_tests/t0_main/lightning_fabric/test_fabric_logging.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
```

Comment:
Having a full integration test is certainly nice to have. But I don't see any other tests here. 

Lightning has a relatively big set of unit tests here:
https://github.com/Lightning-AI/pytorch-lightning/blob/master/tests/tests_pytorch/loggers/test_wandb.py

I think it would be good to adopt these. Otherwise the implementation would easily be subject to regressions.